[DOCS] Adds note on how to update custom rules of Metric anomaly jobs (#249)
elastic/observability-docs: docs/en/observability/inspect-metric-anomalies.asciidoc

By default, the Kubernetes partition field `kubernetes.namespace` is selected.
=====
+
Partitions allow you to create independent models for different groups of data 
that share similar behavior. For example, you may want to build separate models 
for machine type, or cloud availability zone, so that anomalies are not weighted 
equally across groups.
+
5. Click *Enable Jobs*.
6. You're now ready to explore your metric anomalies. Click *View anomalies* to 
view the {ml-docs}/ml-gs-results.html[Anomaly Explorer in Machine Learning].

[NOTE]
=====

> These pre-defined {anomaly-jobs} use {ml-docs}/ml-rules.html[custom rules]. You 
> can update the rules in the *Anomaly explorer* by selecting the *Configure 
> rules* option under *actions*. The changes take effect for new results only. If 

Reviewer: ```suggestion
These pre-defined {anomaly-jobs} use {ml-docs}/ml-rules.html[custom rules]. To
update the rules in the *Anomaly Explorer*, select *actions > Configure 
rules*. The changes only take effect for new results.
```

> you want to apply the changes to existing results, clone and rerun the job.

Reviewer: ```suggestion
If you want to apply the changes to existing results, clone and rerun the job.
```
